browser/ntp_sponsored_images/view_counter_service_factory.cc

```diff
@@ -68,11 +70,8 @@ KeyedService* ViewCounterServiceFactory::BuildServiceInstanceFor(
 }
 
 void ViewCounterServiceFactory::RegisterProfilePrefs(
-    user_prefs::PrefRegistrySyncable* registry) {
-  registry->RegisterBooleanPref(
-      kBrandedWallpaperNotificationDismissed, false);
-  registry->RegisterBooleanPref(
-      kNewTabPageShowBrandedBackgroundImage, true);
+     user_prefs::PrefRegistrySyncable* registry) {
+  RegisterSponsoredImagesProfilePrefs(registry);
```

Comment:
can we use the common pattern of a static `RegisterProfilePrefs` method on the service?

Reply:
done